src/screens/SignInV2/elements/BottomOverlay/index.js

```diff
@@ -113,10 +113,10 @@ const bottomOverlayStyles = StyleSheet.create({
     zIndex: 10
   },
   containerBtnLogin: {
-    backgroundColor: 'white',
+    backgroundColor: COLORS.white,
     borderWidth: 1,
     borderRadius: 8,
-    borderColor: COLORS.lightgrey,
+    borderColor: COLORS.gray300,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here are replacing hard-coded color values with predefined constants which is a good practice for maintainability and consistency across the app. However, the borderColor has been changed from `COLORS.lightgrey` to `COLORS.gray300`. If this change in color was intended, then it's fine. Otherwise, you might want to revert it back to `COLORS.lightgrey`.
```diff
- borderColor: COLORS.gray300,
+ borderColor: COLORS.lightgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->